src/post/router/postsRouter.ts

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+import PostController from "../controller/PostsController/PostsController.js";
+import Post from "../model/Post.js";
+
+const postsRouter = express.Router();
+
+const postController = new PostController(Post);
+
+postsRouter.use("/", postController.getPosts);
```

Comment:
¿Por qué ponéis un controlador para todos los métodos que vayan al path `/`? ¿Cuántos endpoints habéis implementado en una sola línea?

Reply:
Ha sido un error que me he dado cuenta haciendo los test, en la app habia que poner el metodo use y el router ya se encargará de hacer cualquier cosa y aplicar el metodo correspondiente a cada ruta que empieze por `/posts`